Devscord.DiscordFramework.UnitTests/Commands/BotCommandsServiceTests.cs

```diff
@@ -50,7 +51,7 @@ public void ShouldMapToTemplate()
             var template = new BotCommandsTemplateBuilder().GetCommandTemplate(typeof(SmallTestCommand));
             var customTemplate = new Regex(@"run\s*(?<TestUser>\<\@\S+\>)?\s*(?<TestNumber>\d+)", RegexOptions.Compiled | RegexOptions.IgnoreCase);
             var input = "-run <@1234567890> 12";
-            var parsingService = new BotCommandsParsingService(new BotCommandsPropertyConversionService(), new BotCommandsRequestValueGetterService()); //todo mock and test
+            var parsingService = new BotCommandsParsingService(new BotCommandsPropertyConversionService(new UniversalNumberParser()), new BotCommandsRequestValueGetterService()); //todo mock and test
```

Comment:
w testach jednostkowych, tak jak nazwa wskazuje, chcesz być niezależny od zależności
a nie zależny od wszystkiego

chodzi o to że jak odpalasz test to chcesz wiedzieć co konkretnie sie wywaliło
a nie że coś nie działa... i na tym kończą sie informacje

Reply:
Zauważ, że przed moją zmianą kod też nie był niezależny. Ja tutaj tylko dodałem ten krótki zapis, aby to wszystko działało. Byłem zaskoczony, że ten test ma w sobie zależności, ale mnie tylko interesuje tutaj zrobić testy do RunnerOfIBotCommandMethods. 

Poza tym ten kod (nie licząc mojej zmiany) jest **na produkcji,** więc pomyślałem, że musi tak być skoro ty oraz mateusz to przepuściliście na mastera.
**Jeśli ci się nie podoba ten kod to dlaczego to jest na produkcji? Czemu zgodziłeś się na taki kod?**

Wiem, że ten bot ma wiele rzeczy do załatania lub po prostu do zrobienia, ale tego PR robię już bardzo długo i nie ma sensu wymyślać co chwila jakiś konieczności naprawy czegoś co nie jest związane z tym co robię obecnie, bo ten PR będzie się ciągnął w nieskończoność. Jeśli chcesz to mogę kiedyś postarać się tutaj zmienić te testy z BotCommandsServiceTests na odpowiedniejsze, ale może w ramach innego zadania.